spec/features/settings/ext_cert_feature_spec.rb

```diff
@@ -0,0 +1,233 @@
+require "rails_helper"
+
+# rubocop:disable RSpec/ExampleLength
+describe "Feature: External Cerificate settings", js: true do
+
+  let!(:user) { create(:user) }
+  let!(:fixture_path) { RSpec.configuration.fixture_path }
+
+  let(:ssl_cert_file_a) { File.join(fixture_path, "ext_cert_ssl_a.pem") }
```

Comment:
What these different files indicate should be explained in a comment, and something more descriptive than a/b would be good. Like "valid/invalid"

Reply:
Done